src/components/NewTenantHeader.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import BackIcon from "../assets/back.svg?react";
+import { useNavigate } from "react-router-dom";
+
+interface NewTenantHeaderProps {
+  isCompleted: boolean;
+}
+
+const NewTenantHeader: React.FC<NewTenantHeaderProps> = ({ isCompleted }) => {
+  const navigate = useNavigate();
+
+  const handleBackClick = () => {
+    navigate(-1);
+  };
+
+  const progressBarStyle = isCompleted ? "bg-gray-800" : "bg-gray-300";
+  const progressCircleStyle = isCompleted
+    ? " border-gray-800"
+    : "border-gray-300";
+
+  return (
+    <div className="sticky top-0 z-50">
+      <nav className="border-gray-200 bg-white px-2 py-2.5 sm:px-4">
+        <div className="mx-auto flex flex-wrap items-center justify-between">
+          <div className="flex-1 flex justify-between gap-5 py-5 px-52 items-center whitespace-nowrap">
+            <div className="flex gap-5 items-center text-xl font-semibold">
+              <button type="button">
+                <BackIcon onClick={handleBackClick} />
```

Comment:
What happens if a user click from the second page does the color of theprogressBarStyle changes? You have implemented this change only in button click